vllm/core/block_manager_v1.py

```diff
@@ -255,14 +266,35 @@ def __init__(
                 Device.CPU, block_size, num_cpu_blocks)
         # Mapping: seq_id -> BlockTable.
         self.block_tables: Dict[int, BlockTable] = {}
+        # Mapping: req_id -> BlockTable
+        # Note that each SequenceGroup has a unique
+        # request ID
+        self.cross_block_tables: Dict[str, BlockTable] = {}
+
+    def _get_seq_num_required_blocks(self, seq: Sequence) -> int:
+        if seq is None:
+            return 0
+        return len(seq.logical_token_blocks)
```

Comment:
minor simplification suggestion
```suggestion
        return 0 if seq is None else len(seq.logical_token_blocks)
```

Reply:
Thanks! Fixed in most recent push.